.gitignore

```diff
@@ -24,3 +24,5 @@ yarn-error.log*
 # Deployment files
 git-sha
 ga-tag
+
+appregistry.json
```

Comment:
I think this will ignore the file everywhere in the repo, whereas really we just want to ignore the file in its new location at `src/assets/appregistry.json`, right?

Reply:
It does ignore that file everywhere, but I'd wager that's what we want. As in, I don't think we have a reason to have a separate `appregistry.json` somewhere else, and ignoring it everywhere means the `.gitignore` won't change if we decide to change the location again.

Reply:
I think it would be better to be more precise. Then, if you happen to have appregistry.json somewhere where the app is not looking for it, then you will be less confused why your changes aren't appearing, etc. The cost for that is one extra change (to .gitignore) if/when we decide to change where the app does look for this file.